client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -873,31 +873,33 @@ export const fetchLabelsStatus = ( orderId, siteId ) => ( dispatch, getState ) =
 		const setSuccess = json => {
 			response = json.label;
 		};
-		const setIsSaving = saving => {
-			if ( ! saving ) {
-				dispatch( {
-					type: WOOCOMMERCE_SERVICES_SHIPPING_LABEL_STATUS_RESPONSE,
-					orderId,
-					siteId,
-					labelId,
-					response,
-					error,
-				} );
-				if ( error ) {
-					dispatch( NoticeActions.errorNotice(
-						`Failed to retrieve shipping label refund status. ${ error }`,
-						{ id: 'wcs-fetch-label-status' }
-					) );
-				}
+
+		const setIsSaving = () => {
```

Comment:
This function name doesn't make sense anymore. What about `storeResponse`?

Reply:
How about just having it anonymous, inline?